homeassistant/components/tuya/sensor.py

```diff
@@ -138,6 +138,21 @@
         ),
         *BATTERY_SENSORS,
     ),
+    # Irrigator
+    # https://developer.tuya.com/en/docs/iot/categoryggq?id=Kaiuz1qib7z0k
+    "ggq": (
```

Comment:
Missing temperature, humidity, water quantity, rain sensor information, total water consumption?

Reply:
Haven't added this because I cannot test and verify. Mine is just supporting battery and countdown.

Reply:
We can use the docs to add them, and create a virtual device in the cloud platform to test that

Reply:
Are you able to find an Irrigator as virtual device. I cannot find one here: https://iot.tuya.com/pmg/solution

Reply:
😧 What the heck?
Ok, lets implement what you have in this point in that case, but lets add it as a note/comment to the entity descriptions that not all from the spec is implemented.